server/game/event.js

```diff
@@ -27,8 +27,10 @@ class Event {
             return;
         }
 
-        card.markAsSaved();
-        this.cards = _.reject(this.cards, c => c === card);
+        card.game.raiseMergedEvent('onCharacterSaved', { card: card }, () => {
```

Comment:
I would rename this to just `onCardSaved` because we should also have the event fire when saving locations / attachments (even if it won't be used yet). There's little value having two separate events.

You also need to look at raising this when a dupe is used to save a card. There also may be other event handlers that do not use `Event.saveCard` - look for any handlers for onCharacterKilled that do just an `event.cancel()` instead (there may or may not be any)